script.js

```diff
@@ -1,4 +1,125 @@
+// There are 2 players and players take turns.
+// When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+// The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+// After both players have rolled and chosen dice order, the player with the higher combined number wins.
+//--------
+
+//Declare global variables and game mode (2 - one to roll and one to choose order)
+var gameMode = "roll dice"; // initial game mode
+var gameMode_rollDice = "roll dice"; //assign var to different game mode
```

Comment:
Good practice using variables to store string constants! One additional thing to note is to use scream case for these variables--it's a convention. E.g. GAME_MODE_ROLL_DICE